README.md

```diff
@@ -302,14 +302,28 @@ build and install.
 
 # Using the `queue` module
 
+## Initialization
 ```lua
 queue = require 'queue'
 ```
 
 The request "require 'queue'" causes automatic creation of
 the `_queue` space, unless it already exists. The same request
 also sets the necessary space triggers and other objects
-associated with queues.
+associated with queues.<br />
```

Comment:
Just splitting of spaces with an empty line should work for GFM (any markdown dialect in fact).

Reply:
Updated.
https://gist.github.com/shaunlebron/746476e6e7a4d698b373